Tensile/Common.py

```diff
@@ -339,7 +339,7 @@ def getArchitectureName(gfxName):
 
 # These type are newly supported and we would like to use a better file naming for them: _TiToTc_
 # For the rest of the typed, we keep them with old existing naming.
-typesUsingNewNaming = [ ('H','H','S'), ('H','S','S'), ('B','S','S'),('I8','I','I')]
+typesUsingNewNaming = [ ('H','S','S'), ('B','S','S'),('I8','I','I')]   # ('H','H','S') is removed bcs we are merging this case in HBH
```

Comment:
"merging" implies to me that the old HBH functionality still exists. Something like "...bcs it is replacing the legacy HBH type"

Reply:
HBH functionality still exists and this PR fixes its behavior, by not converting the datatype of alpha/beta to fp16.  The proper name for this case is HHS_BH, but due to some naming conflict (as I was told by Chris and Alex), we need to stick to HBH for this case. So, indeed we are merging the HHS_BH in HBH.